app/src/main/java/com/brandon/forzenbook/repository/ForzenRepositoryImpl.kt

```diff
@@ -14,103 +14,71 @@ class ForzenRepositoryImpl(
-                            return LoginToken(token = null, isServiceError = true)
+                            println("Brandon_Test_Data Failed to Convert Response to Entity. ${response.body()}")
+                            return null
                         }
                     } else {
-                        println("Brandon_Test_Data Network No Internet Access.")
-                        return LoginToken(token = null, isNetworkError = true)
+                        println("Brandon_Test_Data Unexpected Response Code ${response.code()}")
+                        return null
                     }
                 } catch (e: Exception) {
-                    println("Brandon_Test_Data Error Deleting Data from Database $e")
-                    return LoginToken(token = null, isDatabaseError = true)
+                    println("Brandon_Test_Data Network Call Failed. $e")
+                    return null
                 }
             }
         } catch (e: Exception) {
             println("Brandon_Test_Data Error Accessing Database to Check for Data $e")
-            return LoginToken(token = null, isDatabaseError = true)
+            return null
         }
         println("Brandon_Test_Data Reached End of Block without Returning.")
-        return LoginToken(token = null, isDatabaseError = true, isServiceError = true)
+        return null
     }
 
-    override suspend fun createUser(createUserData: CreateUserData): CreateUserErrors {
+    override suspend fun createUser(createUserData: CreateUserData): Boolean {
+        val data = createUserData.toCreateAccountRequest()
         try {
-            val data = createUserData.toCreateAccountRequest()
-            if (forzenApiService.isNetworkAvailable(context)) {
-                try {
-                    val response = forzenApiService.createUser(data)
-                    try {
-                        if (response.code() == 200) {
-                            return CreateUserErrors()
-                        }
-                    } catch (e: Exception) {
-                        println("Brandon_Test_Data Unexpected response code. Stating Failure. $e")
-                        return CreateUserErrors(isServiceError = true)
-                    }
-                } catch (e: Exception) {
-                    println("Brandon_Test_Data Failed to call api service. $e")
-                    return CreateUserErrors(isServiceError = true)
-                }
-            } else {
-                println("Brandon_Test_Data No Internet Connection")
-                return CreateUserErrors(isNetworkError = true)
+            val response = forzenApiService.createUser(
+                firstname = data.firstName,
+                lastname = data.lastName,
+                email = data.email,
+                location = data.location,
+                dateOfBirth = data.date,
+            )
+            println(response)
```

Comment:
In production code use Log instead of print for all the prints in your project

Reply:
Updated to use log instead.